torch/_inductor/runtime/triton_helpers.py

```diff
@@ -31,6 +31,21 @@ def _log2(x):
         raise NotImplementedError
 
 
+def set_driver_to_cpu():
+    if backend := triton.backends.backends.get("cpu", None):
+        triton.runtime.driver.set_active(backend.driver())
+        return
+    raise RuntimeError("Could not find an active CPU backend")
```

Comment:
It seems like this is breaking out-of-tree backends like MTIA. It's possible for the user to create a new backend that PyTorch doesn't know about, and register it at runtime. If possible it would be better if we could be agnostic about which backend is in use, as opposed to erroring out if it's not one of N known backends.

Here are a few options that come to mind to work around the issue:
1. Remove the `raise RuntimeError` and make this a warning instead.
2. Refactor this into a method of `TritonScheduling`, so derived backends like MTIA can override it with their own `set_driver` function.
https://github.com/pytorch/pytorch/blob/main/torch/_inductor/codegen/triton.py#L2968

Does either of those sound reasonable?

Reply:
By the way, we should probably add some tests for derived backends to catch this kind of thing in the future. I can look into that if folks agree.

Reply:
Per workchat conversation, I have gone with option #1. Since MTIA doesn't actually run the generated kernels on the CPU, it doesn't matter which active backend Triton has, so there's no need to error out.

Reply:
Sounds good, thanks. Approved.